.gitignore

```diff
@@ -0,0 +1,2 @@
+/*
```

Comment:
Please remove this file from the commit

Reply:
No problem, I taught .gitignore shouldn't commit by default.